dependencies/pom.xml

```diff
@@ -83,7 +83,6 @@
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
-            <version>2.8.0</version>
```

Comment:
We should probably define a specific version here even if it's the latest

Reply:
The version number was removed as per Jeremy's suggestion to allow spring manage the dependency.